### PR TITLE
Add a lot of readme.md files and delete all Thumbs.db files

### PR DESCRIPTION
**Pull request by FlameAlt53**

### CODE REVIEW

**Comment by FlameAlt53** [comment]
you should squash and merge this btw

File: _nds/TWiLightMenu/akmenu/themes/COD Black Ops/readme.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+### COD Black Ops
+
+Author:
```
Comment (NightYoshi370) [inline _nds/TWiLightMenu/akmenu/themes/COD Black Ops/readme.md]: Chadrcheze

File: _nds/TWiLightMenu/akmenu/themes/COD Black Ops/readme.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+### COD Black Ops
+
+Author: 
+
+Date released:
```
Comment (NightYoshi370) [inline _nds/TWiLightMenu/akmenu/themes/COD Black Ops/readme.md]: 5th Aug 2011

File: _nds/TWiLightMenu/akmenu/themes/Cars 2/readme.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+### Cars 2
```
Comment (NightYoshi370) [inline _nds/TWiLightMenu/akmenu/themes/Cars 2/readme.md]: Made on 5th Aug 2011 by  Chadrcheze

File: _nds/TWiLightMenu/akmenu/themes/Knuckles the Echidna/readme.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+### Knuckles the Echidna
```
Comment (NightYoshi370) [inline _nds/TWiLightMenu/akmenu/themes/Knuckles the Echidna/readme.md]: Knuckles the Echidna was made by Biochao on 7th December, 2009

File: _nds/TWiLightMenu/akmenu/themes/Luigi's Mansion/readme.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+### Luigi's Mansion
```
Comment (NightYoshi370) [inline _nds/TWiLightMenu/akmenu/themes/Luigi's Mansion/readme.md]: Made by rockleeace on 31st Dec 2011

File: _nds/TWiLightMenu/akmenu/themes/MKDS Dark/readme.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+### MKDS Dark
```
Comment (NightYoshi370) [inline _nds/TWiLightMenu/akmenu/themes/MKDS Dark/readme.md]: Made by amassenburg on 1st Jan 2010

File: _nds/TWiLightMenu/akmenu/themes/BWBrazil/readme.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+### BWBrazil
```
Comment (NightYoshi370) [inline _nds/TWiLightMenu/akmenu/themes/BWBrazil/readme.md]: By Cannonfoddr on 5th Feb 2010

File: _nds/TWiLightMenu/akmenu/themes/COD Black Ops/readme.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
+### COD Black Ops
```
Comment (NightYoshi370) [inline _nds/TWiLightMenu/akmenu/themes/COD Black Ops/readme.md]: Made by  rockleeace on 23rd Dec 2011

**Pull request merged**